### PR TITLE
Fixes malformed links

### PR DESCRIPTION
**Pull request by sfshaza2**
Fixes malformed links, reported in Issue https://github.com/flutter/website/issues/5387.

### CODE REVIEW

**Review by kwalrath — APPROVED** [review]
lgtm. I happened to notice something questionable about URLs in one page, but it's unrelated to your changes.

**Comment by sfshaza2** [comment]
Fixed.

File: src/docs/perf/faq.md [hunk 1]
```diff
@@ -3,91 +3,123 @@ title: Performance FAQ
...
-  * https://flutter-dashboard.appspot.com/
-  * https://flutter-flutter-perf.skia.org/t/?subset=regressions
-  * https://flutter-engine-perf.skia.org/t/?subset=regressions
-  * https://flutter-dashboard.appspot.com/benchmarks.html (to be retired)
+  * [Flutter dashboard on appspot][]
+  * [Flutter Skia dashboard][]
+  * [Flutter Engine Skia dashboard][]
+
+[Flutter dashboard on appspot]: https://flutter-dashboard.appspot.com/
+[Flutter engine Skia dashboard]: https://flutter-engine-perf.skia.org/t/?subset=regressions
```
Comment (kwalrath) [inline src/docs/perf/faq.md:15]: Uh oh, capitalization looks wrong! (engine vs. Engine)

Comment (kwalrath) [reply src/docs/perf/faq.md:15]: But... I guess it isn't case sensitive? (Seems to be working just fine!)

**Pull request merged**